Cpp2IL.Core/Model/Contexts/MethodAnalysisContext.cs

```diff
@@ -310,6 +310,18 @@ public ConcreteGenericMethodAnalysisContext MakeGenericInstanceMethod(params IEn
         }
     }
 
+    public ConcreteGenericMethodAnalysisContext MakeConcreteGenericMethod(IEnumerable<TypeAnalysisContext> typeGenericParameters, IEnumerable<TypeAnalysisContext> methodGenericParameters)
+    {
+        if (this is ConcreteGenericMethodAnalysisContext)
+        {
+            throw new("This is already concrete generic.");
```

Comment:
Not in love with this exception. The grammar isn't ideal, and we really shouldn't be throwing raw exceptions from API surfaces.
```suggestion
            throw new InvalidOperationException($"Attempted to make a ConcreteGenericMethodAnalysisContext concrete: {this}")
```